templates/templates.go

```diff
@@ -196,15 +196,23 @@ func PkgInfo(w io.Writer, res ddao.GetSingleResultRow) {
 			req, err := http.NewRequestWithContext(ctx, "GET", reqURL, nil)
 			if err != nil {
 				results[index].StatusCode = http.StatusInternalServerError
-				results[index].Error = err.Error()
+				if err == context.DeadlineExceeded {
+					results[index].Error = "Request timed out. The server is taking too long to respond. Use the link above to access original log file."
+				} else {
+					results[index].Error = err.Error()
+				}
 				return
 			}
 
 			// Make the request
 			resp, err := http.DefaultClient.Do(req)
 			if err != nil {
 				results[index].StatusCode = http.StatusInternalServerError
-				results[index].Error = err.Error()
+				if err == context.DeadlineExceeded || err.Error() == "context deadline exceeded" {
```

Comment:
I think that with `errors.Is`, the second clause won't be needed, as this will fire even for a wrapped DeadlineExceeded.